dotcom-rendering/src/client/adaptiveSite.ts

```diff
@@ -16,6 +16,14 @@ export const shouldAdapt = async (): Promise<boolean> => {
 	if (!window.guardian.config.switches.adaptiveSite) return false;
 	if (window.location.host !== 'www.theguardian.com') return false;
 
+	/**
+	 * The europe beta front is being served to a 0% audience. This means it's rarely in cache and so it gets adapted more often.
+	 * This is a temporary measure to ensure that the front is not adapted during testing.
+	 */
+	if (window.guardian.config.tests.europeBetaFrontVariant === 'variant') {
```

Comment:
Do we also want to check that we're specifically on the europe page? From a quick look on my end it seems this would still be true even if someone was on another front. 

<img width="580" alt="image" src="https://github.com/user-attachments/assets/162e3185-de5c-4042-b79d-98ccf15c3277" />

Reply:
If not, or if we're not concerned about that then this looks good!

Reply:
yep, good point! i'll add this in 👍